src/pages/contributors.js

```diff
@@ -0,0 +1,60 @@
+import Head from 'next/head';
+import styles from '../styles/contributors.module.css';
+
+export const getStaticProps = async () => {
+    const res = await fetch('https://api.github.com/repos/WebXDAO/webXDAO.github.io/contributors');

```

Comment:
We can filter out the time using GitHub's graphql API, can you take a look into that?

This will also fix the fact that this is only checking for the repo, not the whole org

Reply:
why I need to add graphql ? this is working perfectly... : ) 

and @mkubdev told me to use the api and fetch the data like this. That's why I did like this 

![image](https://user-images.githubusercontent.com/104062485/193437505-a22e8dc8-f0cc-4630-8acb-fdcd5e840ea7.png)

Reply:
Should be fine using the current method unless there are any differences/improvements using GitHub's GraphQL API.

Reply:
So what should I do now ?

Reply:
@Panquesito7 this will only calculate the contributors of this repo not the org, also it will have all contributors not hacktoberfest contributors

Reply:
@krshkun Hi, yeah the task was to show only contributors from this repos. We can enhance the display with a graphql implementation later in a future feature request! (You can open it!)